pkg/safeguards/tests/unique-service-selectors/USS-error-manifest.yaml

```diff
@@ -1,21 +1,23 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: Service
 metadata:
-  name: my-deployment
-  labels:
-    app: my-app
-    environment: production
-    testLabel3: randomlabel
+  name: my-service-1
 spec:
-  replicas: 3
   selector:
-    matchLabels:
-      app: my-app
-  template:
-    metadata:
-      labels:
-        app: my-app
-    spec:
-      containers:
-        - name: my-container
-          image: nginx:latest
\ No newline at end of file
+    app.kubernetes.io/name: MyApp
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 9376
+---
+apiVersion: apps/v1
+kind: Service
+metadata:
+  name: my-service-2
```

Comment:
you should also explicitly specify the namespace on all the service objects

Reply:
fixed